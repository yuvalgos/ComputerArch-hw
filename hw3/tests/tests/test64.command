./dflow_calc tests/test64.dat tests/test64.in p89 d32 d44 p128 p60 p60 d110 p111 d148 d179 p156 d49 d128 d75 p150 d2 p12 d141 p66 p126 d173 d180 d117 p79 p168 d13 d33 d125 d145 p64 d89 d127 d26 d182 p47 p9 d147 p33 d23 p131 p89 p145 p89 d14 p184 d76 p169 d36 p152 d91 p137 d46 p116 d98 p61 p167 p95 p144 p54 d177 p129 d71 p81 d78 d16 p94 p149 d75 d144 p36 d164 d119 p173 d150 d97 d96 p181 d89 d71 p177 p90 d86 d159 d174 d176 d97 d56 p11 d3 p74 d97 p173 d149 p166 p87 d81 p156 d95 p83 p133 p53 d119 p190 p150 d10 p151 p105 p139 d123 d149 d192 p166 d70 d82 p55 d134 d183 d195 d154 d170 d37 p93 p89 d145 p66 d10 d13 p93 d20 d189 d105 d126 p24 d2 p107 d98 p158 p149 d163 d125 p169 p161 d143