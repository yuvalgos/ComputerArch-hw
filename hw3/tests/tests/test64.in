8 31 3 15
8 23 30 1
4 7 29 17
8 4 12 30
8 5 12 27
0 4 16 9
0 15 14 8
3 14 6 15
1 20 16 7
6 8 4 28
0 15 0 0
5 22 12 18
0 5 29 24
7 31 27 13
8 22 12 8
2 19 1 23
7 23 20 31
2 18 16 8
4 27 8 30
5 11 11 30
2 1 0 8
8 5 30 24
6 31 1 6
8 31 5 10
6 0 5 18
4 7 5 0
3 18 18 12
4 24 0 21
4 6 25 3
7 22 19 17
6 9 25 18
3 28 22 7
6 4 25 19
8 23 24 31
5 26 28 19
8 26 11 15
1 0 30 6
3 31 7 4
8 14 12 26
5 27 4 13
7 10 31 31
7 9 24 16
7 12 3 18
2 14 15 0
6 19 8 11
3 14 2 17
2 24 19 20
4 15 24 3
7 2 18 19
4 13 17 2
2 16 18 16
5 25 23 2
0 29 15 1
4 24 5 25
6 23 22 10
7 21 4 28
6 8 14 16
8 30 21 26
6 21 2 0
7 1 31 26
1 23 6 4
3 6 23 17
7 15 20 16
5 13 29 11
6 26 22 28
2 30 23 8
3 13 25 23
5 18 29 14
7 27 1 14
6 22 31 4
0 30 18 27
4 1 17 25
7 17 24 5
6 28 13 2
1 26 19 8
5 17 28 24
1 4 1 4
7 31 10 9
8 27 21 10
6 31 22 29
3 28 15 20
5 6 29 7
8 2 10 24
7 24 28 29
3 21 8 7
7 16 24 6
3 24 26 18
6 27 28 15
8 15 11 0
8 1 29 28
3 31 22 21
8 7 10 26
4 29 7 21
5 25 24 29
4 7 20 25
1 3 30 16
8 10 11 8
0 7 23 22
5 29 26 6
8 8 31 16
2 17 8 28
0 14 25 9
6 9 10 27
1 29 17 14
0 4 16 16
3 3 29 9
5 27 5 8
5 18 14 26
2 17 5 9
3 29 24 10
3 12 22 25
3 22 6 3
3 12 5 3
1 6 11 24
3 12 28 10
6 9 21 29
2 17 3 14
6 0 10 7
6 30 21 20
3 9 10 12
4 25 0 10
7 11 28 29
5 5 21 26
4 0 25 31
6 7 0 30
2 8 7 8
3 31 15 27
3 16 7 13
5 8 30 18
0 7 30 12
8 14 14 6
2 17 17 16
6 24 11 14
5 29 15 2
0 5 31 28
1 10 31 30
0 30 12 5
6 31 17 4
4 22 4 22
2 2 6 3
0 17 14 25
4 28 19 18
5 13 26 7
1 21 19 31
0 11 6 0
6 30 10 10
4 11 1 26
6 25 8 29
8 31 29 8
0 9 6 2
3 18 11 17
6 25 16 2
2 2 23 18
4 10 3 18
5 14 8 3
4 8 19 17
0 12 23 1
6 12 21 9
0 24 19 18
1 23 31 10
4 31 0 17
0 26 6 4
7 2 7 20
1 6 3 21
0 31 3 1
7 11 26 17
6 29 16 20
4 3 19 2
5 21 23 31
0 2 13 12
2 3 19 5
3 13 12 6
0 9 10 31
1 7 27 20
1 23 4 20
6 25 10 23
3 17 30 14
2 17 15 16
4 2 20 8
0 16 14 27
4 26 13 26
8 5 28 1
3 4 13 0
4 13 5 31
2 2 27 13
3 3 23 19
1 29 29 25
3 0 23 22
2 8 28 6
4 24 16 6
1 8 1 21
1 4 3 12
0 10 11 21
4 7 4 31
6 20 30 25
7 21 28 0
5 14 31 25
6 0 8 21
5 20 2 24
6 24 18 28
