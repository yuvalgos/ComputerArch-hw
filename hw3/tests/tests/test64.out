Reading the opcodes latency info from tests/test64.dat ... Got latency for 9 opcodes
Reading the program file tests/test64.in ... Found 200 instructions
getProgDepth()==212
getDepDepth(89)==62
getInstDeps(32)=={-1,15}
getInstDeps(44)=={9,19}
getDepDepth(128)==140
getDepDepth(60)==45
getDepDepth(60)==45
getInstDeps(110)=={69,93}
getDepDepth(111)==138
getInstDeps(148)=={133,128}
getInstDeps(179)=={154,106}
getDepDepth(156)==85
getInstDeps(49)=={-1,48}
getInstDeps(128)=={118,107}
getInstDeps(75)=={73,53}
getDepDepth(150)==152
getInstDeps(2)=={-1,-1}
getDepDepth(12)==0
getInstDeps(141)=={44,107}
getDepDepth(66)==88
getDepDepth(126)==136
getInstDeps(173)=={106,8}
getInstDeps(180)=={171,161}
getInstDeps(117)=={96,97}
getDepDepth(79)==76
getDepDepth(168)==179
getInstDeps(13)=={-1,-1}
getInstDeps(33)=={27,23}
getInstDeps(125)=={124,99}
getInstDeps(145)=={135,135}
getDepDepth(64)==59
getInstDeps(89)=={52,80}
getInstDeps(127)=={124,66}
getInstDeps(26)=={17,-1}
getInstDeps(182)=={171,123}
getDepDepth(47)==41
getDepDepth(9)==2
getInstDeps(147)=={128,133}
getDepDepth(33)==52
getInstDeps(23)=={21,-1}
getDepDepth(131)==143
getDepDepth(89)==62
getDepDepth(145)==146
getDepDepth(89)==62
getInstDeps(14)=={-1,9}
getDepDepth(184)==185
getInstDeps(76)=={71,32}
getDepDepth(169)==185
getInstDeps(36)=={-1,28}
getDepDepth(152)==159
getInstDeps(91)=={40,74}
getDepDepth(137)==148
getInstDeps(46)=={44,8}
getDepDepth(116)==138
getInstDeps(98)=={74,81}
getDepDepth(61)==48
getDepDepth(167)==104
getDepDepth(95)==100
getDepDepth(144)==117
getDepDepth(54)==78
getInstDeps(177)=={88,127}
getDepDepth(129)==83
getInstDeps(71)=={-1,51}
getDepDepth(81)==54
getInstDeps(78)=={58,40}
getInstDeps(16)=={8,13}
getDepDepth(94)==125
getDepDepth(149)==143
getInstDeps(75)=={73,53}
getInstDeps(144)=={113,123}
getDepDepth(36)==11
getInstDeps(164)=={105,89}
getInstDeps(119)=={96,114}
getDepDepth(173)==136
getInstDeps(150)=={146,140}
getInstDeps(97)=={60,69}
getInstDeps(96)=={19,56}
getDepDepth(181)==151
getInstDeps(89)=={52,80}
getInstDeps(71)=={-1,51}
getDepDepth(177)==141
getDepDepth(90)==76
getInstDeps(86)=={74,67}
getInstDeps(159)=={148,153}
getInstDeps(174)=={104,8}
getInstDeps(176)=={145,154}
getInstDeps(97)=={60,69}
getInstDeps(56)=={45,50}
getDepDepth(11)==0
getInstDeps(3)=={-1,-1}
getDepDepth(74)==53
getInstDeps(97)=={60,69}
getDepDepth(173)==136
getInstDeps(149)=={113,139}
getDepDepth(166)==141
getDepDepth(87)==62
getInstDeps(81)=={52,25}
getDepDepth(156)==85
getInstDeps(95)=={70,85}
getDepDepth(83)==62
getDepDepth(133)==122
getDepDepth(53)==88
getInstDeps(119)=={96,114}
getDepDepth(190)==192
getDepDepth(150)==152
getInstDeps(10)=={-1,-1}
getDepDepth(151)==143
getDepDepth(105)==131
getDepDepth(139)==138
getInstDeps(123)=={120,90}
getInstDeps(149)=={113,139}
getInstDeps(192)=={165,168}
getDepDepth(166)==141
getInstDeps(70)=={67,68}
getInstDeps(82)=={40,53}
getDepDepth(55)==59
getInstDeps(134)=={126,80}
getInstDeps(183)=={181,164}
getInstDeps(195)=={141,187}
getInstDeps(154)=={128,105}
getInstDeps(170)=={44,134}
getInstDeps(37)=={25,32}
getDepDepth(93)==112
getDepDepth(89)==62
getInstDeps(145)=={135,135}
getDepDepth(66)==88
getInstDeps(10)=={-1,-1}
getInstDeps(13)=={-1,-1}
getDepDepth(93)==112
getInstDeps(20)=={-1,9}
getInstDeps(189)=={179,163}
getInstDeps(105)=={103,102}
getInstDeps(126)=={88,106}
getDepDepth(24)==24
getInstDeps(2)=={-1,-1}
getDepDepth(107)==127
getInstDeps(98)=={74,81}
getDepDepth(158)==159
getDepDepth(149)==143
getInstDeps(163)=={105,143}
getInstDeps(125)=={124,99}
getDepDepth(169)==185
getDepDepth(161)==102
getInstDeps(143)=={44,137}
